pkg/azuredisk/azuredisk_v2.go

```diff
@@ -89,6 +92,31 @@ const (
 	azDriverNodeHealthy      azDriverNodeStatus = "Driver node healthy."
 )
 
+// Metrics subsystem and all of the keys used by the rest client.
+const (
+	RestClientSubsystem = "kube_client"
```

Comment:
You have these constants defined in two files. We don't need these here if we are defining them in `azure_constants.go`. The naming foe these ones is probably better tho.